test-providers.d/io-github-autotest-libvirt.ini

```diff
@@ -1,5 +1,5 @@
 [provider]
-uri: https://github.com/autotest/tp-libvirt.git
+uri: file:///root/tp-libvirt/
```

Comment:
why do we change this ?

Reply:
sorry, it is the wrong modification, I will update soon, thanks